quick-start/bucket/Kptfile

```diff
@@ -0,0 +1,46 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: bucket-demo
```

Comment:
Blueprint name should match directory name (`bucket`) and generally we should avoid "demo" blueprints.